burn-wgpu/src/ops/base.rs

```diff
@@ -1,8 +1,14 @@
-use std::{marker::PhantomData, sync::Arc};
-
+use crate::{
+    context::WorkGroup,
+    element::WGPUElement,
+    kernel::{build_info, KernelSettings},
+    kernel_wgsl,
+    pool::get_context,
+    tensor::WgpuTensor,
+    GraphicsAPI, WGPUDevice,
```

Comment:
Do you want to also change these to CamelCase so that we are consistent across the board?

Here is the list: 
 - WGPUElement -> WgpuElement
 - GraphicsAPI -> GraphicsApi
 - WGPUDevice -> WgpuDevice